docs/conf/portland/2025/news/announcing-cfp.rst

```diff
@@ -0,0 +1,70 @@
+:template: {{year}}/generic.html
+:banner: _static/conf/images/headers/portland-2024-speakers.jpg
+:og:image: _static/conf/images/headers/portland-2024-speakers.jpg
+
+.. post:: Oct 28, 2024
+   :tags: {{shortcode}}-{{year}}, website, cfp, tickets
+
+Announcing the Call for Proposals and Sponsorship Prospectus
+============================================================
+
+We're excited to announce our Portland `Call for Proposals <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/cfp/>`_, opening even earlier than last year, even though the conference back at the start of May this time around.
```

Comment:
```suggestion
We're excited to announce our Portland `Call for Proposals <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/cfp/>`_, opening even earlier than last year, even though the conference is back at the start of May this year.
```

Reply:
There are lots of words here that are tripping me up with possible double meanings ( `conference is back at the start...` etc). Also, I'm not sure folks will notice/fully appreciate the relative date shifts (while acknowledging here that it is notable :) ). Does the wider delta give submitters additional time to work on proposals or to polish them once approved?

In any case, just suggestions:

```suggestion
We're excited to announce our Portland `Call for Proposals <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/cfp/>`_, which is opening earlier this year than last year, giving you __[more time to blah blah]__ since the conference is in May instead of April this year.
```
or just:
We're excited to announce our Portland `Call for Proposals <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/cfp/>`_, which is opening earlier this year!